neqo-common/src/fuzz.rs

```diff
@@ -0,0 +1,45 @@
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+use std::{
+    collections::hash_map::DefaultHasher,
+    fs::File,
+    hash::{Hash, Hasher},
+    io::Write,
+    path::Path,
+};
+
+/// Write a data item `data` for the fuzzing target `target` to the fuzzing corpus. The caller needs
+/// to make sure that `target` is the correct fuzzing target name for the data written.
+///
+/// # Panics
+///
+/// Panics if the corpus directory does not exist or if the corpus item cannot be written.
+pub fn write_item_to_fuzzing_corpus(target: &str, data: &[u8]) {
+    // This bakes in the assumption that we're executing in the root of the neqo workspace.
+    // Unfortuntely, `cargo fuzz` doesn't provide a way to learn the location of the corpus
+    // directory. As a safety measure, panic if the directory does not exist.
+    let corpus = Path::new("../fuzz/corpus").join(target);
+    assert!(
+        corpus.is_dir(),
+        "corpus directory {} does not exist",
+        corpus.display()
+    );
```

Comment:
Can you make it instead?

Reply:
I can, but that would take away this being a safety measure.